### PR TITLE
1.13/1764 design changes

### PR DESCRIPTION
**Pull request by Nikomus**
https://trello.com/c/BfOlDp2u/1764-design-changes-v-ee

Individual changes listed per commit

### CODE REVIEW

File: resources/views/store/edit_registration.blade.php [hunk 1]
```diff
@@ -180,47 +180,43 @@ class="@if($errors->has('pri_carer'))invalid @endif" type="text"
...
                         <i class="fa fa-ticket button-icon" aria-hidden="true"></i>Go to voucher manager
                     </div>
                 </a>
-            </div>
-        </form>
+                
+                @if (!isset($registration->family->leaving_on) )
 
-        @if (!isset($registration->family->leaving_on) )
-            <form action="{{ URL::route('store.registration.family',['registration' => $registration]) }}" method="post"
```
Comment (charlesstrange2) [inline resources/views/store/edit_registration.blade.php:197]: This form hit a different endpoint?

**Review by charlesstrange2 — CHANGES_REQUESTED** [review]
check the old leaving form - it looks like a different endpoint

File: resources/views/store/edit_registration.blade.php (hunk 1 above)
Comment (Nikomus) [reply resources/views/store/edit_registration.blade.php:197]: I think my formaction went missing with some markup shuffling

Comment (charlesstrange2) [reply resources/views/store/edit_registration.blade.php:197]: Ah! I see you've merged the two forms and then got the second button to submit all the formdata to the update route too?

Comment (charlesstrange2) [reply resources/views/store/edit_registration.blade.php:197]: is that why we do Ah! Is that why formNoValidate?

Comment (charlesstrange2) [reply resources/views/store/edit_registration.blade.php:197]: hmm. this ticket says we support ie11 😨 which formNoValidate isn't a thing for 😬 

Should we have the two forms back?

**Review by charlesstrange2 — CHANGES_REQUESTED** [review]
reinstate 2 forms?

**Review by charlesstrange2 — APPROVED** [review]
okedoke :+1:

**Pull request merged**